python/paddle/fluid/framework.py

```diff
@@ -2996,14 +2996,31 @@ def find_name(var_list, name):
                 for m in proto.outputs:
                     if (m.name not in outputs) and m.dispensable:
                         continue
-                    if not ((m.name in outputs) or m.dispensable):
-                        raise ValueError(
-                            (
-                                "Incorrect setting for output(s) of "
-                                "operator \"%s\", should set: [%s]."
+
+                    # Primitive mode currently doesn't support intermediate output
```

Comment:
Why not support

Reply:
reshape primitive currently doesn't set xshape output, and this will be fixed in a new pr.